src/components/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+// eslint-disable-next-line import/no-unresolved
+import '/src/assets/scss/elements/_login.scss';
```

Comment:
It is necessary to include it in the import in the file index.scss, and remove this line from here. Also, the directory must be specified relative to the root (for example, "@import './assets/scss/elements/_login_page .scss';", And remove line "// eslint-disable-next-line import/no-unresolved"